### DISCUSSION TITLE
Is it expected that an empty .nextflow/plr is created in the current dir when calling nextflow pull?

### DISCUSSION THREAD

**Discussion by SamStudio8** [nextflow-io/nextflow#3941] [Q&A]
Recently I've noticed that using `nextflow pull` will create an empty `.nextflow/plr` directory in the current working directory. I normally associate this directory with local plugins downloaded when running a workflow, not pulling one. This behaviour appears to have been added in 22.08.0-edge; it it expected?

**Comment by pditommaso** [comment]
It maybe be a side effect of [this change](https://github.com/nextflow-io/nextflow/commit/80fba6e9). More specifically, the adding of the Plugins.init() here 

https://github.com/nextflow-io/nextflow/blob/master/modules/nextflow/src/main/groovy/nextflow/cli/CmdPull.groovy#L73-L73

I guess you would like to avoid the creation of that dir with the pull command.

**Reply by SamStudio8** [reply]
Thanks for confirming! Yes, it would be useful for `nextflow pull` to avoid unexpected changes to the CWD 😇
